pages/15.Plugins/08.Hubspot/docs.en.md

```diff
@@ -11,24 +11,27 @@ taxonomy:
 
 -------------------
 
-## Mautic - HubSpot CRM plugin
+## Mautic - Hubspot CRM plugin
 
-[Mautic][mautic] can push contacts to [HubSpot CRM][Hubspot-crm] based on [Contact actions][testing] or [Point Triggers][points].
+[Mautic][mautic] can push contacts to [Hubspot CRM][Hubspot-crm] based on [Contact actions][testing] or [Point Triggers][points].
 
-## HubSpot API key
+## Hubspot API key
 
-1. Create a [HubSpot CRM][Hubspot-crm] account if you don't have one already.
+1. Create a [Hubspot CRM][Hubspot-crm] account if you don't have one already.
 
-1. Visit [https://app.hubspot.com/api-key][hubspot-keys] to generate your [HubSpot API key][hubspot-keys].
+2. Create a private app to [get credentials by following the Hubspot documentation][https://developers.hubspot.com/docs/api/private-apps]. You need to give to your private app the adapted rights for Webmecanik to handle contacts, companies, etc.
```

Comment:
What are the adapted rights for Webmecanik? I guess this is supposed to say: give your private app the adapted rights for your use case. So it has access to the needed contacts, companies, etc.###

Reply:
Bad copy paste :D